spec/v3/services/owner/find_spec.rb

```diff
@@ -93,7 +93,8 @@
             "name"            => "master"},
           "starred"           => false,
           "managed_by_installation"=>false,
-          "active_on_org"     =>nil
+          "active_on_org"     => nil,
+          "migration_status"  => nil
```

Comment:
```suggestion
          "migration_status"  => nil,
```

I like putting in trailing commas at the end; makes future diffs a tiny bit cleaner. YMMV